src/clj/domain/spec.clj

```diff
@@ -101,15 +101,15 @@
 (s/def :course/question-sets (s/coll-of ::question-set))
 
 
-(s/def ::course
+(s/def :course/course
```

Comment:
Why did you turn this into a simple kw? It's kinda weird with the duplication. We would have to do that for all specs aswell, i.e. ::user, ::course-iteration etc.

Reply:
Using `::` makes this a "local" spec (:: will be replaced with the current namespace when reading the file) and explicitly declaring the namespace of the keyword makes it easily accessible from the outside (for example tests or domain functions).

I have no idea why the parts of the entities are "public" while the aggregate is "local".

Reply:
I think because they didn't want to use the doubled keyword. I'll make this an issue.

Reply:
#186 #187